### PR TITLE
provide prerequisites setup example for ubuntu

### PR DESCRIPTION
**Pull request by ghost**
make setup a bit quicker

## Test

validated on a fresh ubuntu~22.04~20.04 instance

### CODE REVIEW

**Comment by GuybrushX** [comment]
FYI: Currently only Ubuntu 18.04 and 20.04 are officially supported. Using anything else isn't recommended for production use so you're on your own if something's going wrong. 20.04 is recommended.

**Comment by ghost** [comment]
> FYI: Currently only Ubuntu 18.04 and 20.04 are officially supported

ty, switched to 20.04

File: README.md [hunk 1]
```diff
@@ -49,6 +49,15 @@ The Casper MainNet is live.
 * g++
 * recommended [wasm-strip](https://github.com/WebAssembly/wabt) (used to reduce the size of compiled Wasm)
 
+```sh
+# Ubuntu prerequisites setup example
+apt update
+apt install cmake libssl-dev pkg-config gcc g++ -y
+# the '-s -- -y' part ensures silent mode. Ommit if you want to customize
```
Comment (Fraser999) [inline README.md]:
```suggestion
# the '-s -- -y' part ensures silent mode. Omit if you want to customize
```

**Review by Fraser999 — COMMENTED** [review]
One typo, but otherwise LGTM.

**Review by Fraser999 — APPROVED** [review]

**Comment by ghost** [comment]
@Fraser999 , be aware that for some reason, authorship was not kept here by the merge.

(Edit: Just so you know, no problem from my side)

**Pull request merged**